pom.xml

```diff
@@ -65,7 +65,7 @@
     <dependency>
       <groupId>org.mirah</groupId>
       <artifactId>mirah-complete</artifactId>
-      <version>0.1.1</version>
+      <version>0.1.3</version>
```

Comment:
Mirah 0.1.4  version could be set in plugin configuration. I'm using following plugin config with 0.1.4 published in my local artifactory:
```xml
<plugin>
                    <groupId>org.mirah.maven</groupId>
                    <artifactId>maven-mirah-plugin</artifactId>        
                    <version>1.2-SNAPSHOT</version>				  
                    <executions>
                        <execution>
                            <id>mirah-compile</id>
                            <phase>compile</phase>
                            <goals>
                                <goal>compile</goal>
                            </goals>
                        </execution>
                        <execution>
                            <id>mirah-test-compile</id>
                            <phase>test-compile</phase>
                            <goals>
                                <goal>testCompile</goal>
                            </goals>
                        </execution>
                    </executions>
                    <configuration>
                        <verbose>false</verbose>
                        <newClosures>false</newClosures>
                    </configuration>
					<dependencies>
					  <dependency>
						<groupId>org.mirah</groupId>
						<artifactId>mirah-complete</artifactId>
						<version>0.1.4</version>
					  </dependency>
					</dependencies>
                </plugin>
```